azure-devops/enforce-clang-format.cmd

```diff
@@ -9,6 +9,8 @@ tools/inc ^
 tools/jobify/jobify.cpp ^
 tools/parallelize/parallelize.cpp ^
 tools/validate/validate.cpp
-@echo If your build fails here, you need to format the following files with
+@echo If your build fails here, you need to format the following files with:

```

Comment:
should this read "with clang format version: \r\n @clang-format.exe --version"

Reply:
oh wait, echo is on.

Reply:
The following line already says `@clang-format.exe --version` so we're suppressing the echo. However, it prints `clang-format version 9.0.0 (tags/RELEASE_900/final)` so it reads nicely.

Reply:
ah I see.